src/python/pants/backend/go/util_rules/build_pkg_target.py

```diff
@@ -78,6 +85,9 @@ class BuildGoPackageTargetRequest(EngineAwareParameter):
     # If True, then force coverage instead of applying import path patterns from `build_opts.coverage_config`.
     with_coverage: bool = False
 
+    # Extra dependencies to force on the target. Useful for implicit linker dependencies.
+    extra_dependencies: tuple[Address, ...] = ()
```

Comment:
why not
```python
extra_dependencies: Addresses = ()
```
?

Reply:
Because it is not a pattern I've ever used so not in mind when I wrote this code.